registry/server/templates/routes/getRenderedTemplate.ts

```diff
@@ -28,13 +29,30 @@ const validateRequestBeforeGetTemplateRendered = validateRequestFactory([{
 }]);
 
 async function getRenderedTemplate(req: Request<GetTemplateRenderedRequestParams>, res: Response): Promise<void> {
+    let defaultTemplate;
+
     const {
         name: templateName,
     } = req.params;
 
-    const { locale } = req.query;
+    const { locale, domain } = req.query;
+
+    if (domain) {
+        const [domainItem] = await db.select('id').from<RouterDomains>('router_domains').where('domainName', domain as string);
+
+        if (domainItem) {
```

Comment:
add please comment - why `domainItem` can be empty. and is it expected behaviour or we just mute error?

Reply:
Because of provided domain name might not be found.